.github/workflows/codeql.yml

```diff
@@ -0,0 +1,35 @@
+name: "CodeQL"
+on:
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['main']
+  schedule:
+  - cron: '54 3 * * 2'
```

Comment:
The default was something like `39 3 ** 2`, I think, the whole config is pretty much default without comments. 

I shifted the time a bit so we don't have contention on GHA runners with other repos in IBM org if everyone else would keep default. As for particular time I made it 5-4-3-2 just for fun 😆

Reply:
Do we even need a schedule? Isn't running on push/PR sufficient for our needds?

Reply:
@ricellis It was like that in the default template and I was pondering on "why" too. My guess is that since CodeQL works by parsing code into a special kind of AST database and then running bunch of predefined queries over it the scheduler here to address a case where some new threats been added into those queries packs, but code hasn't been touched for a while. Essentially to have some trigger to detect newly found vulns in existing code, rather than new degradation in the changed code.

Reply:
yeah, that was my best guess too.